core/authentication_api.php

```diff
@@ -798,8 +811,8 @@ function auth_process_plain_password( $p_password, $p_salt = null, $p_method = n
  * @return string 16 character random password
```

Comment:
Update phpdoc to add `@param` for new parameter and update `@return` comment.  Also update the description of the function to reflect that it is not always 16 characters.

Reply:
Changed.